jdaviz/app.py

```diff
@@ -575,7 +575,7 @@ def get_viewer_by_id(self, vid):
         """Like :meth:`get_viewer` but use ID instead of reference name.
         This is useful when reference name is `None`.
         """
-        return self._viewer_store[vid]
+        return self._viewer_store.get(vid)
```

Comment:
This behavior is now inconsistent with that of `def _viewer_by_reference(self, reference)`. Won't it be too confusing if accessing it one way silently gives `None` but another will still throw exception?

Reply:
@pllim is this comment still of concern?

Reply:
I guess not.